src/extension/flutter/flutter_daemon.ts

```diff
@@ -50,6 +51,7 @@ export class FlutterDaemon extends StdIOService<UnknownNotification> implements
 
 		if (portFromLocalExtension) {
 			this.createNcProcess(portFromLocalExtension);
+			this.startPing();
 		} else if (workspaceContext.config.forceFlutterWorkspace && config.daemonPort) {
 			this.createNcProcess(config.daemonPort);
```

Comment:
Do we need to startPing() in this branch?

Reply:
I think this won't be used very much any more, but it would make sense to add it here too.